cogs/fishing.py

```diff
@@ -290,30 +289,36 @@ async def rename(self, ctx: commands.Context, old: str, new: str):
         """
         Renames your fish.
         """
-
-
+        
```

Comment:
Smh set up your VSC to trim trailing spaces.

Reply:
I did this in GitHub 😳